src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs

```diff
@@ -412,12 +412,9 @@ public async Task RemoveAccountAsync(ApplicationConfiguration appConfig, IAccoun
                     }
                     else
                     {
-                        _logger?.WarningPii(
+                        _logger?.ErrorPii(
```

Comment:
Since we do not care, if the account is not found, it should be just a warning and not error.

Reply:
Also, my thoughts are on the border of warning/error. Leaning towards warning.

Reply:
Agreed. We do sometimes have customers opening bugs if they discover error messages in their logs. So unless it is truly an exceptional case, we should log this as a warning.

Reply:
fixed